scripts/calc_rrups_single.sl

```diff
@@ -39,7 +39,7 @@ then
     start_time=`date +${runtime_fmt}`
     echo "python $gmsim/workflow/scripts/management/update_mgmt_db.py $MGMT_DB_LOC rrup running --run_name $REL_NAME --j $SLURM_JOBID" >> ${MGMT_DB_LOC}/mgmt_db_queue/${timestamp}\_${SLURM_JOBID}
 
-    time python ${IMPATH}/calculate_rrups.py -fd ${FD} -np ${SLURM_CPUS_PER_TASK} -o ${OUT_DIR}/rrup_${REL_NAME}.csv ${STATION_FILE} ${SRF_FILE}
+    time python ${IMPATH}/calculate_rrups.py -fd ${FD} -o ${OUT_DIR}/rrup_${REL_NAME}.csv ${STATION_FILE} ${SRF_FILE}
```

Comment:
Does it automatically know how many processes to run on?

Reply:
The argument doesn't exist anymore.

Reply:
It uses all the processes available